kroxylicious-app/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,35 @@
+#
```

Comment:
I'd really like to ditch the properties format at this point as well and switch to the XML version. Feel less prone to configuration errors and is more often cited in Log4j2 examples (AFICT)

Reply:
Yeahp, sounds like a plan. I saw they support YAML which would make all our config the same format, but if XML is going to be the most common way in docs/community we should go that way.

Reply:
I'd have a preference for yaml tbh.  It'll play nicer with the modern eco-system should we ever need anything to programmatically create the log properties file.   I guess  too that avoid having XML parsers on the classpath (fewer vulnerabilities).

Reply:
Lo4j2 has an optional dependencies on:

-  `jackson-dataformat-yaml` for YAML support
- `jackson-databind` for json support
- `jackson-dataformat-xml`  for XML layouts and receivers (?)

So from a security perspective they all amount to Jackson and thus the same profile I think. Log4J recommends one of XML, YAML or JSON over properties as those three are hierarchical like its config model (lets ignore the inception, the config model is hierarchical because its expressed in XML).  

The XML format seems to be the default in all their docs but given they are meant to be equivalent I'm not really fussed which of those three formats we merge with.